src/opera/util/input_validation.py

```diff
@@ -307,23 +307,30 @@ def validate_disp_inputs(runconfig, logger, name):
     dyn_anc_file_group = runconfig.sas_config['dynamic_ancillary_file_group']
     static_anc_file_group = runconfig.sas_config['static_ancillary_file_group']
 
-    check_input_list(input_file_group['cslc_file_list'], logger, name,
-                     valid_extensions=('.h5',), check_zero_size=True)
+    if runconfig.pge_name == 'DISP_S1_PGE':
```

Comment:
Approach looks good here, but the docstring should be updated to indicate that this function is now intended for use with two PGEs

Reply:
Docstring updated